### ISSUE TITLE
Error when running tests on OS X

### ISSUE THREAD

**Issue by waywardmonkeys** [dylan-lang/testworks#41]
On OS X, I do:

```
./Bootstrap.3/bin/libraries-test-suite-app \
  --skip-test=common-extensions-protocol-classes-test \
  --skip-suite=semaphores-suite
```

And after running and while printing the results, it crashes:

```
libraries-test-suite FAILED in 80.406335 seconds:
  Ran 85 suites: 45 passed (52.941176%), 40 failed, 1 skipped, 0 not implemented, 0 crashed
  Ran 294 tests: 260 passed (88.435376%), 25 failed, 1 skipped, 7 not implemented, 2 crashed
  Ran 20044 checks: 19961 passed (99.585904%), 71 failed, 0 skipped, 0 not implemented, 12 crashed
#f is not of type {<class>: <buffer>}
#f is not of type {class <buffer> 0x472ce8}
```

**Comment by pedro-w** [comment]
See https://github.com/pedro-w/opendylan/commit/d2c84efc7b224b7296e7f0498262e3ec5669a1ef

**Comment by cgay** [comment]
@pedro-w did your change above ever get merged into master?

**Comment by pedro-w** [comment]
I checked, it was included in [PR #1107](https://github.com/dylan-lang/opendylan/pull/1107)

The commit referenced above was on a branch with a lot of other things so I re-arranged them into separate branches before putting in the requests.

**Comment by cgay** [comment]
I'm going to close this then. Not having a Mac handy for Dylan work I'm going somewhat on faith. Please re-open if you disagree.
